src/main/java/org/galatea/starter/domain/IexHistoricalPrice.java

```diff
@@ -0,0 +1,20 @@
+package org.galatea.starter.domain;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.util.Date;
+
+@Data
+@Builder
+public class IexHistoricalPrice {
+
+  private String symbol;
+  private Date date;
```

Comment:
Try using classes from the Java 8 datetime API instead of the older java.util.Date. The new classes are better in a lot of ways so it would be good to get a bit more familiar.